nuxeo-importer-core/pom.xml

```diff
@@ -27,13 +27,41 @@
     </dependency>
     <dependency>
       <groupId>org.nuxeo.ecm.platform</groupId>
+      <artifactId>nuxeo-platform-webapp-base</artifactId>
```

Comment:
You can't depend on JSF dependencies in a core module. Furthermore, tests succeed without JSF dpeendencies.